test/test_integration.py

```diff
@@ -106,14 +106,18 @@ def test_start_with_custom_styles():
                       if v['Type'] == "volume" and
                       v['Name'] == "orderly_web_css"][0]
         assert css_volume['Name'] == "orderly_web_css"
-        assert css_volume['Destination'] == "/static/public"
+        assert css_volume['Destination'] == "/static/public/css"
 
         # check that the style files have been compiled with the custom vars
         web_container = cfg.get_container("web")
         style = string_from_container(web_container,
                                       "/static/public/css/style.css")
         assert "/* Example custom config */" in style
 
+        # check that js files are there also
```

Comment:
Unrelated bug that I found - js files were being overwritten by the mounted style volume.